introduction.md

```diff
@@ -0,0 +1 @@
+Oyasumi nasai. My name is Koichi. I follow the Ruby path.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.